glyphs-reader/src/font.rs

```diff
@@ -314,6 +316,51 @@ impl RawGlyph {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Hash)]
+pub struct Instance {
+    pub name: String,
+    pub active: bool,
+    pub type_: InstanceType,
+    pub axis_mappings: BTreeMap<String, RawUserToDesignMapping>,
+}
+
+/// <https://github.com/googlefonts/glyphsLib/blob/6f243c1f732ea1092717918d0328f3b5303ffe56/Lib/glyphsLib/classes.py#L150>
+#[derive(Clone, Debug, PartialEq, Hash)]
+pub enum InstanceType {
+    Single,
+    Variable,
+}
+
+impl From<&str> for InstanceType {
+    fn from(value: &str) -> Self {
+        if value.to_ascii_lowercase() == "variable" {
+            InstanceType::Variable
+        } else {
+            InstanceType::Single
+        }
+    }
+}
+
+#[derive(Debug, Clone, FromPlist, PartialEq, Eq, Hash)]
+struct RawInstance {
+    pub name: String,
+    pub exports: Option<i64>,
+    pub active: Option<i64>,
+    pub type_: Option<String>,
+    pub axes_values: Option<Vec<OrderedFloat<f64>>>,
```

Comment:
Correct. It would be really nice if FromPlist could deserialize to an empty vec but I don't want to tackle that in this PR. 

Filed as https://github.com/googlefonts/fontmake-rs/issues/257